Jack Bienvenue Classnotes Contribution

jackbienvenue opened the pull request:
Preliminary version of my course notes contribution, adapted from my topic presentation on visualizing spatial data with Google Maps. Note that I was unable to render the course notes before initiating this pull request because of a path issue in one of the other files.

Commit:
Edit _quarto.yml to include name and reference to .qmd

Commit:
Addition of quarto document and first draft of notes

Commit:
Addition of images directory, spatial viz subdirectory, and spatial viz images

Commit:
Change instance of Images to images

Commit:
add gmplot and googlemaps to requirements.txt

jun-yan committed:
corrected path to nyccrashes data

jun-yan committed:
incorporated _importExprot.qmd with necessary packages

jun-yan committed:
presentation schedule adjusted given the data science panel date

jun-yan committed:
added preparation for logistic fitting

jun-yan committed:
emily back to 10/28

jun-yan committed:
more explanation to the one hot encoding example

jun-yan committed:
emily back to 10/30

jun-yan committed:
fixed missing blankline to restore wish list

oliviakashalapov committed:
Add topic for presentation (SMOTE) - Olivia Kashalapov

jun-yan committed:
added an overview chapter on ml

jun-yan committed:
removed bold facing in stats.qmd

jun-yan committed:
smoothed tree; added nb

jun-yan committed:
one liner to midterm project in preliminary

jun-yan committed:
added empty _spatialvisual.qmd

jun-yan reviewed (changes requested):
+ No tracking .DS_Store.
+ Every reference listed needs to have been cited in the text.
+ The level of sectioning needs to correction: This is a section (##) within a chapter (#). I have set up a _spatialvisual.qmd file for you to fill. See _plotnine.qmd for example.
+ Can some images be replaced with html gadget?

jun-yan committed:
cleaned references in _plotnine.qmd

jun-yan committed:
use apa bib style

jun-yan committed:
fixed communication citations

jun-yan committed:
updated presentation topics

jun-yan committed:
place holder for smote

jun-yan committed:
added supervised.qmd

jun-yan committed:
placeholder for unsupervised learning and k-means

jun-yan committed:
added boosted trees

Commit:
Remove .DS_Store binary from images folder which was erroneously tracked

Commit:
Make additions into the _spatialvisual.qmd file, small grammar fixes, addition of citations in text

jackbienvenue committed:
Merge branch 'main' into JBclassnotes

jackbienvenue commented:
Hi Dr. Yan!
I removed the .DS_Store binary, added my content to the new .qmd file you made (including fixing sectioning levels), and added references in the text body. I cannot include the html widgets without including my API key in the shared source though to my knowledge, so I did not make any images into widgets. Thanks.

jun-yan commented:
Will make a fresh pull request.
